Modules/Segmentation/LevelSets/include/itkParallelSparseFieldLevelSetImageFilter.hxx

```diff
@@ -841,7 +841,7 @@ ParallelSparseFieldLevelSetImageFilter<TInputImage, TOutputImage>::ThreadedAlloc
 
   // Local histogram for every thread (used during Iterate() )
   m_Data[ThreadId].m_ZHistogram = new int[m_ZSize];
-  for (i = 0; i < m_ZSize; ++i)
+  for (unsigned int i = 0; i < m_ZSize; ++i)
   {
     m_Data[ThreadId].m_ZHistogram[i] = 0;
   }
```

Comment:
Thanks Hans! By the way, we might as well remove this `for` loop, and value-initialize the histogram array elements by `()`:

```cpp
   m_Data[ThreadId].m_ZHistogram = new int[m_ZSize]();
```

Reply:
I did add the performance improvement in the next patch.  I hope it does not delay this patch set any longer.